apps/verticalhome/js/clock.js

```diff
@@ -45,18 +113,18 @@ function Clock() {
    *        tick, should accept a date object as its only argument.
    * @memberOf Clock
    */
-  this.start = function cl_start(refresh) {
+  this.start = function cl_start() {
```

Comment:
as soon as you are changing function's signature, you shall update it's description. By the way, is it intended to stop passing callback as a parameter?